File: Belajar-array/Arraydinamic-standar/list.vb
Imports System
Imports System.Collections.Generic

Module Program
Sub Main()
	Dim buah As New List (Of String) From {"Apel","Pisang"}
	Dim sayur As New List (Of String)()
	sayur.Add("Brokoli")
	buah.Add("Mangga")
	Dim length As Integer = buah.Count
	Dim ambil As String = buah(length-1)
	buah.RemoveAt(length-1)
	Console.WriteLine("ambil " & ambil)
	Console.WriteLine("buah 1:" & buah(0))
	Console.WriteLine("buah 2:" & buah(1))
	Console.WriteLine("sayur 1:" & sayur(0))
End Sub
End Module

File: Belajar-array/Arraydinamic-standar/list.cs
using System;
using System.Collections.Generic;
class Program {
	static void Main() {
		List<string> buah = new List<string>{"Apel","Pisang"};
		List<string> sayur = new List<string>();
		sayur.Add("Brokoli");
		buah.Add("Mangga");
		int len = buah.Count;
		var ambil = buah[len-1];
		buah.RemoveAt(len-1);
		Console.WriteLine("ambil {0}",ambil);
		Console.WriteLine("buah 1:{0}",buah[0]);
		Console.WriteLine("buah 2:{0}",buah[1]);
		Console.WriteLine("sayur 1:{0}",sayur[0]);
	}
}
